### PR TITLE
Mona 137 Fix Lint and console errors.

### PR DESCRIPTION
**Pull request by acsearles**
Hey @mstone6769, can you look over this quickly and make sure I didn't royally screw something up in an effort to fix lint problems. I'm thinking specifically around the cyclonic complexity problems and the regex line length stuff.

### CODE REVIEW

**Comment by acsearles** [comment]
Oh lord, I just looked at the diffs. Nevermind, you'll never be able to see the changes I made. I'll try to fix that.

**Comment by mstone6769** [comment]
allright

File: lib/dist/iui-module.js [hunk 1]
```diff
@@ -72,13 +72,1404 @@
...
+        scope.onSelect = function ($item) {
+          ngModel.$render = angular.noop;
+          ngModel.$setViewValue($item.item);
+          scope.itemSelected = true;
+          if (scope.field) {
+            scope.$emit(scope.field.name + 'Selected', $item.item);
+          }
+          scope.innerModel = $item.key;
+          scope.field.itemSelected = true;
+          timeout(function () {
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: Questionable timeout

File: lib/dist/iui-module.js [hunk 2]
```diff
@@ -72,13 +72,1404 @@
...
+
+        ngModel.$render = function () {
+          scope.selected = ngModel.$modelValue;
+        };
+
+        console.log('HEllo');
+
+        scope.$watch('selected', function (newVal) {
+          ngModel.$setViewValue(newVal);
+          $rootScope.$broadcast(scope.field.name + 'Checked', scope.model);
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: Using $rootScope to broadcast is good in this instance

File: lib/dist/iui-module.js [hunk 3]
```diff
@@ -72,13 +72,1404 @@
...
+    return {
+      restrict: 'E',
+      require: 'ngModel',
+      templateUrl: '/$iui-form/iui-form-fields/iui-date-dropdown/iui-date-dropdown.html',
+      scope: {
+        form: '=',
+        field: '='
+      },
+      link: function link(scope, element, attributes, ngModel) {
+        var moment = require('moment');
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: May want to list moment as a dependency

File: lib/dist/iui-module.js [hunk 4]
```diff
@@ -72,13 +72,1404 @@
...
+            }
+          } else {
+            scope.isValid = false;
+            ngModel.$setViewValue(undefined);
+          }
+        };
+      }
+    };
+  }]);
+  function isLeapYear(year) {
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: Surely moment can do this too

File: lib/dist/iui-module.js [hunk 5]
```diff
@@ -72,13 +72,1404 @@
...
+          ngModel.$setViewValue(scope.dateVal);
+        };
+        scope.format = 'MMMM d, yyyy';
+        scope.maxDate = new Date();
+
+        /* function to open datepicker icon when clicked on it */
+        scope.open = function ($event) {
+          $event.preventDefault();
+          $event.stopPropagation();
+          PopupManager.togglePopup(scope.field.id);
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: Not sure how we'll handle this

**Pull request closed without merging**